inference-engine/ie_bridges/java/cpp/tensor_desc.cpp

```diff
@@ -0,0 +1,223 @@
+#include <inference_engine.hpp>
+
+#include "openvino_java.hpp"
+#include "enum_mapping.hpp"
+#include "jni_common.hpp"
+
+using namespace InferenceEngine;
+
+JNIEXPORT jlong JNICALL Java_org_intel_openvino_TensorDesc_GetTensorDesc(JNIEnv *env, jobject obj, jint precision, jintArray dims, jint layout)
+{
+    static const char method_name[] = "GetTensorDesc";
+    try
+    {
+        auto n_precision = precision_map.find(precision);
+
+        if (n_precision == precision_map.end())
+            throw std::runtime_error("No such precision value!");
+
+        auto n_layout = layout_map.find(layout);
+
+        if (n_layout == layout_map.end())
+            throw std::runtime_error("No such layout value!");
+
+        const jsize length = env->GetArrayLength(dims);
+        jint *data = env->GetIntArrayElements(dims, 0);
+
+        TensorDesc *tDesc = new TensorDesc(n_precision->second, std::vector<size_t>(data, data + length), n_layout->second);
+
+        env->ReleaseIntArrayElements(dims, data, 0);
+
+        return (jlong)tDesc;
+    } catch (const std::exception &e)
+    {
+        throwJavaException(env, &e, method_name);
+    }
+    catch (...)
+    {
+        throwJavaException(env, 0, method_name);
+    }
+    return 0;
+}
+
+JNIEXPORT void JNICALL Java_org_intel_openvino_TensorDesc_setDims(JNIEnv *env, jobject, jlong addr, jintArray dims){
+    static const char method_name[] = "setDims";
+    try
+    {
+        TensorDesc *tDesc = (TensorDesc *)addr;
+
+        const jsize length = env->GetArrayLength(dims);
+        jint *data = env->GetIntArrayElements(dims, 0);
+
+        tDesc->setDims(std::vector<size_t>(data, data + length));  
```

Comment:
Please add a method which creates `std::vector<size_t>` from `jintArray` to jni_common - there are at least 5 usages.